RecoParticleFlow/PFTracking/plugins/GoodSeedProducer.cc

```diff
@@ -222,11 +220,11 @@ GoodSeedProducer::produce(Event& iEvent, const EventSetup& iSetup)
 	  int ipteta=getBin(Tk[i].eta(),Tk[i].pt());
 	  int ibin=ipteta*9;
 	  
-	  float oPTOB=1.f/Tj[i].lastMeasurement().updatedState().globalMomentum().mag();
+	  float oPTOB=1.f/std::sqrt(Tk[i].innerMomentum().mag2()); // FIXME the original code was buggy should be outerMomentum...
```

Comment:
I'm confused: is the current change from ``` .lastMeasurement().updatedState().globalMomentum() ```
to ``` Tk[i].innerMomentum() ``` supposed to preserve identity?

If the goal is to match tracks to calo clusters, then outer momentum makes more sense.
For ID purposes Eout/Pin is more relevant.